packages/compartment-mapper/test/capture-lite.test.js

```diff
@@ -87,3 +91,53 @@ test('captureFromMap() should round-trip sources based on parsers', async t => {
   );
   t.is(actual, expected, 'Source code should not be pre-compiled');
 });
+
+test('captureFromMap() should retain additional module locations', async t => {
+  const readPowers = makeReadPowers({ fs, url });
+
+  const entryModuleLocation = new URL(
+    'fixtures-additional-modules/node_modules/goofy/index.js',
+    import.meta.url,
+  ).href;
+
+  const additionalModuleLocations = {
+    [entryModuleLocation]: [
+      new URL('fixtures-additional-modules/config.js', import.meta.url).href,
+    ],
+  };
```

Comment:
the fixture here mimics something like webpack.  we have a tool (`goofy`) and that tool dynamically loads a config file (`config.js`) from our application root.

Reply:
This test fixture was dropped in lieu of the one created in the target branch.